### PR TITLE
Adfly bot

### PR DESCRIPTION
**Pull request by JordinaGR**
Hi, 

This is an Adfly bot, it enters the link you want (line 7) undefined times. 
I made this with python 3.8 and selenium using the chrome web driver. To be able to run it you should do this:
1. pip install selenium.
2. install the webdriver with this: https://sites.google.com/a/chromium.org/chromedriver/downloads
3. make sure you save it in your python directory.

Thanks!

### CODE REVIEW

File: Adfly/adfly_bot.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from selenium import webdriver
+import time
+
+def link():
+
+    driver.switch_to.window(driver.window_handles[1])
+    driver.get("http://raboninco.com/23079033/proves-boto-groc")
```
Comment (JordinaGR) [inline Adfly/adfly_bot.py:7]: Here you can put your adfly shortened link.

**Review by Boskicl — COMMENTED** [review]
This bot I am building is not made to take people to websites where they will get malware. That link you provided is NOT SAFE. Please do not try to merge something which is harmful to users on the internet. This will never be allowed on my Github or to any developers Github.

**Comment by Boskicl** [comment]
Source who tried to provide merge is not trusted, tried to add a malware link to code.

File: Adfly/adfly_bot.py (hunk 1 above)
Comment (Osunkuwara) [reply Adfly/adfly_bot.py:7]: http://lyksoomu.com/xl2U
http://lyksoomu.com/xl2V
http://lyksoomu.com/xl2W
http://lyksoomu.com/xl2X
http://lyksoomu.com/xl2Y
http://lyksoomu.com/xl2Z
http://lyksoomu.com/xl2a
http://lyksoomu.com/xl2b
http://lyksoomu.com/xl2c
http://lyksoomu.com/xl2d
http://lyksoomu.com/xl2e
http://lyksoomu.com/xl2f
http://lyksoomu.com/xl2g
http://lyksoomu.com/xl2h
http://lyksoomu.com/xl2i
http://lyksoomu.com/xl2j
http://lyksoomu.com/xl2k
http://lyksoomu.com/xl2l
http://lyksoomu.com/xl2m
http://lyksoomu.com/xl2n

**Pull request closed without merging**